### ISSUE TITLE
web plugin querystring search doesn't behave as documented

### ISSUE THREAD

**Issue by clach04** [beetbox/beets#4393]
### Problem

Using the web plugin to search does not yield the same results as `beets ls`.

For example:

	# beet ls avril 14
	Aphex Twin - Drukqs (US Release) - Avril 14th (LP Version)
	# beet ls 14 avril
	Aphex Twin - Drukqs (US Release) - Avril 14th (LP Version)

Entering the same strings into the search form in the web browser appears to generate the following URLs:

    http://HOSTNAME:8337/#item/query/14%20avril
    "GET /item/query/14,avril HTTP/1.1" 200 

NOTE the comma.

This yields no results.

According to https://github.com/beetbox/beets/blob/master/docs/plugins/web.rst#get-itemqueryquerystring two search words would be "AND"'d together (see https://beets.readthedocs.io/en/stable/reference/query.html).

At this point I'm not sure if this is intended or not.

I'm also not clear if the issue is the frontend where it generates the GET request or the web service.

If anyone can update this with some thoughts I can dig into the code a little.

### Setup

beets version 1.6.0
Python version 3.8.4
plugins: play, web

Can  provide config file if really needed.

Thanks for making Beets available, its working great for casting to my Google Chromecast/Speaker!

**Comment by wisp3rwind** [comment]
From https://beets.readthedocs.io/en/stable/reference/query.html

> Note that the comma has to be followed by a space (e.g., foo,bar will be treated as a single keyword, not as an OR-query).

So maybe the frontend is indeed generating the wrong request? But how come this has not been noticed before?

**Comment by sampsyo** [comment]
Very tricky! I'm not entirely sure what's going on here, but it sure seems related to #4182. Do you know if you're running the latest source version of beets? That might be worth doing to confirm—if my guess is right, that PR fixed this issue.

**Comment by clach04** [comment]
@sampsyo  I was definitely not at headsrevs, I'd installed via pip and pypi.

I took your advice and installed latest using:

    pip install git+https://github.com/beetbox/beets.git

I manually viewed and confirmed that beetsplug/web/__init__.py contained the update to https://github.com/beetbox/beets/blob/master/beetsplug/web/__init__.py#L264 - so I'm at headrevs as of a few minutes ago.

Ran the same test (in a chrome based browser), and got the same result :-(

Is anyone NOT seeing the behavior I'm seeing? Be good to collect versions and browsers information that are being used.
